Jenkinsfile

```diff
@@ -1,150 +1,140 @@
 max_time = 180
 
+setup_pip_venv = """
+    rm -rf venv
+    conda list
+
+    python3 -m venv venv
+    source venv/bin/activate
+    python3 -m pip install -U pip setuptools wheel
+
+    python3 -m pip install 'graphviz'
+    python3 -m pip install 'jupyter-sphinx>=0.2.2'
+    python3 -m pip install 'portalocker'
+    python3 -m pip install 'scipy>=1.3.3,<1.5.0'
+    python3 -m pip install 'nose'
+    python3 -m pip install 'docutils'
+    python3 -m pip install 'mu-notedown'
+    python3 -m pip install 'flake8'
+    python3 -m pip install 'awscli>=1.18.140'
+
+    export MPLBACKEND=Agg
+"""
+
+setup_mxnet_gpu = """
+    python3 -m pip install mxnet-cu101==1.7.0
+    export LD_LIBRARY_PATH=/usr/local/cuda-10.1/lib64
+    export MXNET_CUDNN_AUTOTUNE_DEFAULT=0
+"""
+
+cleanup_venv = """
+    deactivate
+    rm -rf venv
+"""
+
 stage("Unit Test") {
   parallel 'core': {
     node('linux-cpu') {
-      ws('workspace/autogluon-core-py3') {
+      ws('workspace/autogluon-core-py3-v3') {
```

Comment:
Why `-v3`?

Reply:
just some unique namespace not used before (-v1/v2 are already used)